src/olympia/reviewers/templates/reviewers/includes/files_view.html

```diff
@@ -11,38 +11,26 @@
     </div>
     <a href="{{ url('devhub.file_validation', addon.slug, file[0].id) }}">{{ _('Validation') }}</a>
     &middot;
-    <a href="{{ url('files.list', file[0].id) }}">{{ _('Contents') }}</a>
+    <a 
+      href="{{ code_manager_url('browse', addon.pk, version.pk) }}"
+      title="Browse all files in this version"
```

Comment:
We likely want to localize this string so it should probably be something like:

```suggestion
      title="{{ _('Browse all files in this version') }}"
```